src/Files.App/Helpers/ContextFlyoutItemHelper.cs

```diff
@@ -565,14 +563,10 @@ private static bool Check(ContextMenuFlyoutItemViewModel item, CurrentInstanceVi
 					CommandParameter = itemViewModel?.CurrentFolder,
 					ShowItem = itemViewModel?.CurrentFolder is not null && (App.DrivesManager.Drives.FirstOrDefault(x => string.Equals(x.Path, itemViewModel?.CurrentFolder.ItemPath))?.MenuOptions.ShowFormatDrive ?? false),
 				},
-				new ContextMenuFlyoutItemViewModel()
+				new ContextMenuFlyoutItemViewModel(commands.EmptyRecycleBin)
 				{
-					Text = "BaseLayoutContextFlyoutEmptyRecycleBin/Text".GetLocalizedResource(),
-					Glyph = "\uEF88",
-					GlyphFontFamilyName = "RecycleBinIcons",
-					Command = commandsViewModel.EmptyRecycleBinCommand,
+					IsEnabled = RecycleBinHelpers.RecycleBinHasItems(),
 					ShowItem = !itemsSelected && currentInstanceViewModel.IsPageTypeRecycleBin,
-					ShowInRecycleBin = true,
```

Comment:
Why was this removed?

Reply:
ShowInThing properties are set to true in the constructor. The ShowItem property has the final say on whether the item is displayed or not. Here it is enough since ShowItem tests IsPageTypeRecycleBin. With contexts, ShowItem will no longer be needed because ShowItem is initialized with IsExecutable in the constructor.